### DISCUSSION TITLE
Delete Restic Data Manually

### DISCUSSION THREAD

**Discussion by whitepiratebaku** [vmware-tanzu/velero#3335] [Community support Q&A] [answered]
**Deleted Backup but Restic data did not get deleted**
I have deleted a backup (used default-volumes-to-restic when created) but restic data did not deleted itself.According to other thread restic does it periodically and delete data.I wonder is there a way to delete restic data manually and normally.

**Comment by dsu-igeek** [comment, marked as answer]
You can remove data from the bucket but I would be very careful doing this because it is quite easy to break backups you may wish to keep.  The safest way to do this would be to use a new bucket and then delete the old bucket.
